modules/ROOT/pages/manage-monitor-applications.adoc

```diff
@@ -2,6 +2,9 @@
 
 Applications and API gateways deployed to Runtime Fabric include native support for Anypoint Monitoring to allow viewing metrics within Anypoint Platform.
 
+[NOTE]
+Monitoring solutions other than Anypoint Monitoring are not supported.
```

Comment:
Actually, it's "is not supported."

Reply:
Hmmm, I think it's OK - the 'are' relates to 'Monitoring solutions'.